mlir/lib/Dialect/Linalg/Utils/Utils.cpp

```diff
@@ -1088,6 +1088,114 @@ bool isaConvolutionOpOfType<linalg::Conv3DOp>(LinalgOp op,
       .matchBody();
 }
 
+// #inputMap  = affine_map<(N, D, H, W, F, d, h, w, c)
+//                    -> (N, D + d, H + h, W + w, c)>
+// #filterMap = affine_map<(N, D, H, W, F, d, h, w, c) -> (d, h, w, c, F)>
+// #outputMap = affine_map<(N, D, H, W, F, d, h, w, c) -> (N, D, H, W, F)>
```

Comment:
I did not point this out in previous reviews. It is optional. I think we can drop the comments because the code self-documents this. We can keep them for consistency.

```
      .matchMaps({/*inputMap=*/{N, m.strided(D, d, 0), m.strided(H, h, 1),
                                m.strided(W, w, 2), c},
                  /*filterMap=*/{d, h, w, c, F},
                  /*outputMap=*/{N, D, H, W, F}})
```